src/common/HTTPRequest.ts

```diff
@@ -160,6 +178,118 @@ export class HTTPRequest {
     return this._url;
   }
 
+  /**
+   * @returns the `ContinueRequestOverrides` that will be used
+   * if the interception is allowed to continue (ie, `abort()` and
+   * `respond()` aren't called).
+   */
+  continueRequestOverrides(): ContinueRequestOverrides {
+    assert(this._allowInterception, 'Request Interception is not enabled!');
+    return this._continueRequestOverrides;
+  }
+
+  /**
+   * @returns the `ResponseForRequest` that will be used
+   * if the interception is allowed to respond (ie, `abort()` is not
+   * called).
+   */
+  responseForRequest(): Partial<ResponseForRequest> {
+    assert(this._allowInterception, 'Request Interception is not enabled!');
+    return this._responseForRequest;
+  }
+
+  /**
+   * @returns the most recent reason for aborting the request
+   */
+  abortErrorReason(): Protocol.Network.ErrorReason {
+    assert(this._allowInterception, 'Request Interception is not enabled!');
+    return this._abortErrorReason;
+  }
+
+  /**
+   * @returns `true` if `continue` is currently the highest priority intercept resolution
+   */
+  private shouldContinue(): boolean {
+    if (!this._allowInterception) return false;
+    if (this._interceptionHandled) return false;
+    if (!this._continueRequested) return false;
+    if (this._abortRequested && this._abortPriority >= this._continuePriority)
+      return false;
+    if (
+      this._respondRequested &&
+      this._respondPriority >= this._continuePriority
+    )
+      return false;
+    return true;
+  }
+
+  /**
+   * @returns `true` if `respond` is currently the highest priority intercept resolution
+   */
+  private shouldRespond(): boolean {
+    if (!this._allowInterception) return false;
+    if (this._interceptionHandled) return false;
+    if (!this._respondRequested) return false;
+    if (
+      this._continueRequested &&
+      this._continuePriority > this._respondPriority
```

Comment:
All other comparisons use `>=` but this one uses `>` - is there a specific reason for that?

Reply:
Continue us the lowest priority. In the event of a tie, respond should win.

If abortPriority <= respondPriority, abort

If respondPriority <= continuePrioirty, respond